src/Tokenizer/Transformer/ArraySquareBraceTransformer.php

```diff
@@ -40,6 +40,16 @@ public function getCustomTokenNames()
     /**
      * {@inheritdoc}
      */
+    public function getRequiredPhpVersionId()
+    {
+        // short array syntax was introduced in PHP 5.4, but the fixer is smart
+        // enough to handel it even before 5.4
+        return 50000;
```

Comment:
maybe we could define the min. PHP version of the Fixer (5.3.6) on the Application class and return that as default on an AbstractTransformer class?